src/Gladiators.Tests/Tests.cs

```diff
@@ -0,0 +1,76 @@
+using System;
```

Comment:
Ces tests ne sont pas des Tests unitaires mais des tests d'intégrations.
cf requirements : Your code should be unit tested (100% Coverage)

Reply:
Ah ok, je ne connaissais pas la différence!